ADO.NET/OAuthSampleApp/README.md

```diff
@@ -2,28 +2,37 @@
 
 The Vertica ADO.NET driver now allows connecting through OAuth access tokens. 
 
-This sample application was built in .NET 6.0 and can be run in .NET Framework 4.6+ or .NET Core 3.1+.
+This application simulates what you would want your application to do when connecting with access and refresh tokens. We store the tokens as environment variables but they can be stored in any fashion. The logic of the program is:
+1. If the access token is set, attempt to connect.
+2. If the access token is not set, and there is a refresh token, attempt to get a new access token.
```

Comment:
nit: first comma is unnecessary. Two independent clauses joined by and does not need a comma. Come on now Kevin, we live in a society with grammar rules.

Reply:
I fixed it. Begrudgingly